pkg/cmd/log.go

```diff
@@ -0,0 +1,150 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/fntlnz/kubectl-trace/pkg/factory"
+	"github.com/fntlnz/kubectl-trace/pkg/logs"
+	"github.com/fntlnz/kubectl-trace/pkg/meta"
+	"github.com/fntlnz/kubectl-trace/pkg/signals"
+	"github.com/fntlnz/kubectl-trace/pkg/tracejob"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	batchv1client "k8s.io/client-go/kubernetes/typed/batch/v1"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+)
+
+var (
+	logShort = `` // Wrap with i18n.T()
+	logLong  = logShort + `
+
+...`
+
+	logExamples = `
+  # ...
+  %[1]s trace log -h
+
+  # ...
+  %[1]s trace log`
+)
+
+// LogOptions ...
+type LogOptions struct {
+	genericclioptions.IOStreams
+	traceID      *types.UID
```

Comment:
Can we just visually separate (with a blank line) those that are flags from those that are arguments? :)

Eg., 
```go
type LogOptions struct {
   genericclioptions.IOStreams
   ...
   ...
   traceID
}
```